.github/workflows/run-tests.yml

```diff
@@ -21,6 +21,7 @@ on:
   push:
     branches:
       - main
+      - fix_isodate_tests
```

Comment:
```suggestion
```

Just so we don't forget that

Reply:
lemme plug that in once we're done-done with this